### DISCUSSION TITLE
Mobile app in Samsung DEX - keyboard handling

### DISCUSSION THREAD

**Discussion by FueLee** [community/community#34718] [Mobile]
Hi,
When I use my Samsung phone in DEX station and want to approve a Github Login event with Github mobile app, I can not use the enter key on keyboard to approve the digits in the "New Signin Request" window. 

I can enter the digits with the keyboard (external USB to DEX), but the Enter does not trigger the "Approve" button, instead it is evaluated as a new line character and the cursor goes down by 1 line.

Could you check it?
Thanks

**Comment by HenSquared** [comment]
Hi @FueLee,

Thanks for the feedback. Sorry to hear this is blocking your workflow. I've opened up an issue with the team to track this. I will reach out again here if I have other questions. thanks!
